pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "ohsome"
-version = "0.1.0b1"
+version = "0.1.0b4"
```

Comment:
I guess 0.1.0b4 was set to push it to test pypi. The latest version on pypi is 0.1.0b1 though. So this needs to be changed to 0.1.0b2 before it's pushed to pypi. But publishing to pypi is not done automatically through github yet anyways, right?

Reply:
yep. before the merge in master we'll also have to clean all the 0.1.0b1+ releases from test.pypi or else the flow will fail.